### PR TITLE
fix(client): code blocks overflowing parents

### LINKED ISSUE #41418: Code Block Styling Escapes Parent
Page with issue: https://www.freecodecamp.org/learn/data-visualization/data-visualization-projects/visualize-data-with-a-bar-chart

I think there were some recent CSS changes, and they might have caused this:
image

I am going to open this up to first timers, as it should be a small change to a `code` element within the `#description` element.

An issue with the _help wanted_ or _first timers only_ label is open for contribution. The first comprehensive PR created will be reviewed and merged. We typically do not assign issues to anyone other than long-time contributors.

If you would like to contribute, and have not read the contributors docs, please do so here: https://contribute.freecodecamp.org/#/

If you have any issues with contributing, be sure to join us on the [contributors channel](https://chat.freecodecamp.org/channel/contributors), or on the [contributors sub-forum](https://forum.freecodecamp.org/c/contributors/3)

### PR DESCRIPTION
**Pull request by PatrickFranz**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #41418

### CODE REVIEW

File: client/src/components/layouts/prism.css [hunk 1]
```diff
@@ -7,6 +7,11 @@ pre[class*='language-'] {
   background: var(--primary-background);
 }
 
+pre[class*='language'] code[class*='language-'] {
+  overflow-wrap: anywhere;
+  white-space: pre-wrap;
+}
+
```
Comment (nhcarrigan) [inline client/src/components/layouts/prism.css:14]:
```suggestion
pre[class*='language'] code[class*='language-'] {
  white-space: nowrap;
  overflow: auto;
}

.video-quiz-option {
  max-width: 90%;
}
```

I spoke with @ahmadabdolsaheb and it does sound like horizontal scroll is the desired approach.

This CSS change will preserve the horizontal scroll that exists on the code within a video challenge question, while adding it to the video challenge answer. It also shrinks the answer text width a bit to account for the select button.

Certainly open to better CSS approaches for this same effect, though 🙂

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

**Comment by ahmadabdolsaheb** [comment]
The original issue seems to persist.
Screen Shot 2021-03-16 at 1 19 23 PM

**Comment by raisedadead** [comment]
Hi @ahmadabdolsaheb since the linked issue is for a first timer could you guide with the exact changes that need to be done in the corresponding files.

**Comment by ahmadabdolsaheb** [comment]
@PatrickFranz, thanks for your patience. It seem the issue is more involved than initially expected. I will be pushing additional changes to resolve the issue across our platform upcoming days.

**Comment by raisedadead** [comment]
Thanks for the clarification @ahmadabdolsaheb in that case do you mind approving the current changes to get this PR moving forward.

**Comment by PatrickFranz** [comment]
> The original issue seems to persist.
> Screen Shot 2021-03-16 at 1 19 23 PM

Do you have a URL for this? I'd like to see why that code block isn't affected by the rule change.

**Comment by PatrickFranz** [comment]
OK, so not sure what to do here. Should I close it? Keep waiting? @ahmadabdolsaheb could you provide some clear direction on what we're doing with this?

**Review by naomi-lgbt — APPROVED** [review]
Hey @PatrickFranz, 

Thanks for your patience on this. Your changes look good to me, and the initial issue is resolved on my end:
image


I do think we will need to circle back and style this scrollbar, but that can be done in a separate PR at a later time.

image

This looks good to me.

**Review by raisedadead — APPROVED** [review]

**Comment by nhcarrigan** [comment]
Congratulations on making your first pull request to our project. We have reviewed your contributions and are happy to accept them.

We hope you enjoyed the contribution process, and encourage you to watch for issues labelled `help wanted` for additional opportunities to contribute.

**Pull request merged**